StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSession.swift

```diff
@@ -77,6 +99,8 @@ public extension StripeAPI {
         public let livemode: Bool
         @_spi(STP) public let paymentAccount: PaymentAccount?
         @_spi(STP) public let bankAccountToken: BankAccountToken?
+        let status: Status?
+        let statusDetails: StatusDetails?
```

Comment:
backend model to double-check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/resource/link_account_session_resource.rb#L91-106